indexer/Cargo.toml

```diff
@@ -33,9 +33,9 @@ backoff = { version = "0.4.0", features = ["tokio"] }
 
 [dependencies.hub-core]
 package = "holaplex-hub-core"
-version = "0.5.0"
+version = "0.5.5"
 git = "https://github.com/holaplex/hub-core"
-branch = "stable"
+features = ["kafka_internal"]
```

Comment:
Is this released?

Reply:
oh so thats why im not able to use the stable branch.
we def need a release of hub-core then